drakrun/drakrun/scripts/cfg.template

```diff
@@ -24,3 +24,4 @@ cpuid="host,htt=0"
 vga="stdvga"
 vif = [ 'type=ioemu,model=e1000,bridge=drak{{ VM_ID }}' ]
 disk = [ {{ DISKS }} ]
+processor_trace_buf_kb = 65536
```

Comment:
Isn't it going to cause errors when running stock Xen? Are there checks for unknown config options?

Reply:
afaik this is no-op for stock Xen

Reply:
:+1: